agent/cloudinit/file_writer.go

```diff
@@ -90,7 +90,7 @@ func (w FileWriter) WriteToFile(file *Files) error {
 		ownerFormatLen := 2
 
 		if len(owner) != ownerFormatLen {
-			return errors.Wrap(err, fmt.Sprintf("Invalid owner format '%s'", file.Owner))
+			return fmt.Errorf("invalid owner format '%s'", file.Owner)
```

Comment:
Any specific reason for this change? As I see Wrap is used at L77 as well, so we should be consistent about this change.

Reply:
This was a bug which surfaced when we added test for this check. There is no error before this to wrap it.